lineman/app/components/legacy_trial_expired_modal/legacy_trial_expired.coffee

```diff
@@ -0,0 +1,23 @@
+shownToGroup = {}
+angular.module('loomioApp').factory 'LegacyTrialExpiredModal', (ModalService, AppConfig)  ->
+
+  appropriateToShow: (group, user) ->
+    !shownToGroup[group.id] and
```

Comment:
This user is always CurrentUser, yeah?

I think this wants to live in something like `AbilityService.canSeeExpiryModalFor(group)`?